src/widget/wlibrarytableview.h

```diff
@@ -42,6 +55,8 @@ class WLibraryTableView : public QTableView, public virtual LibraryView {
     void loadVScrollBarPosState();
     void saveVScrollBarPosState();
 
+    QMap<QString, int> m_vScrollBarPosValues;
```

Comment:
Why do you use QString instead of TrackModel* as your key? Pointers are valid key values.